server/s3_motr_kvs_writer.cc

```diff
@@ -517,7 +517,7 @@ void S3MotrKVSWriter::put_keyval(
   writer_context->init_kvs_write_op_ctx(kv_list.size());
 
   // Ret code can be ignored as its already handled in async case.
-  put_keyval_impl(kv_list, true, false, 0, 0);
+  put_keyval_impl(kv_list, true, false, 0, 0, STABLE);
```

Comment:
This flag should come from put_keyval() itself

Reply:
In put_keyval_impl you have to set the call back
idx_op_ctx->cbs->oop_executed = your eecuted callback